mcs/class/System.XML/Dummy.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Xml.Schema;
+
+namespace System.Xml
```

Comment:
You didn't notice that those in SR.cs are in different namespace. Have you ever tried to actually remove those my classes and build? It won't. There is reason they are there.